Sources/Screens/RepositoryDetail/RepositoryDetailView.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+struct RepositoryDetailView: View {
+    @StateObject
+    var viewModel: RepositoryListView.ViewModel
```

Comment:
It is not a good idea to reuse the same ViewModel on different screens. I can give you many reasons why in your particular case, but I will give you a few:
1. You don't need searchTerm in RepositoryDetailView
2. You will not need a timer here which will publish some changes every second here.
3. You don't need an array of repositories here.

Instead you can create a new ViewModel for this view and put the repository @ line 6 into it.